libshaderc_spvc/src/spvc.cc

```diff
@@ -89,6 +89,33 @@ void shaderc_spvc_compile_options_set_output_language_version(
   options->glsl.version = version;
 }
 
+void shaderc_spvc_compile_options_set_shader_model(
+    shaderc_spvc_compile_options_t options, uint32_t model) {
+  options->hlsl.shader_model = model;
+}
+
+void shaderc_spvc_compile_options_set_fixup_clipspace(
+    shaderc_spvc_compile_options_t options, bool b) {
+  options->glsl.vertex.fixup_clipspace = b;
+}
+
+void shaderc_spvc_compile_options_set_flip_vert_y(
+    shaderc_spvc_compile_options_t options, bool b) {
+  options->glsl.vertex.flip_vert_y = b;
+}
+
+// Fill options with given data.  Return amount of data used, or zero
+// if not enough data was given.
+size_t shaderc_spvc_compile_options_set_for_fuzzing(
+    shaderc_spvc_compile_options_t options, const uint8_t *data, size_t size) {
+    if (!data || size < sizeof(*options))
+      return 0;
+
+    size_t used = std::min(size, sizeof(*options));
```

Comment:
The call to std::min is not needed here because you check that `size` is not smaller that `sizeof(*options)` above.